recipe/meta.yaml

```diff
@@ -20,6 +20,7 @@ requirements:
     - python
   run:
     - fastcache
+    - setuptools
```

Comment:
SymPy should not be runtime depending on pkg_resources.

Reply:
Interesting. The CIs failed though without it :-/
I'll investigate this later. (Unless you beat me to it.)

Reply:
OK, I see why this is happening. We changed our `isympy` script to install as a setuptools entrypoint if setuptools is installed (I think it was needed for Windows). https://github.com/sympy/sympy/pull/13193. So this is an upstream problem.